examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
For papers that are closer to the heart of VAEs: 
Might be useful to cite Kingma's [IAF paper](https://arxiv.org/abs/1606.04934).
Larsen's [vae-gan paper](https://arxiv.org/abs/1512.09300) is good as well.